packages/node_modules/samples/browser-meetings/app.js

```diff
@@ -0,0 +1,99 @@
+/* eslint-env browser */
+
+/* global Webex */
+
+/* eslint-disable require-jsdoc */
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-console */
+/* eslint-disable no-global-assign */
+/* eslint-disable no-multi-assign */
+
+// Globals
+let webex;
+
+// Collapsible Setup
+const collapsibles = document.querySelectorAll('.collapsible');
+
+function collapse() {
+  const prevState = this.getAttribute('data-collapsed');
+  const newState = prevState === 'true' ? 'false' : 'true';
+
+  this.setAttribute('data-collapsed', newState);
+  this.nextElementSibling.setAttribute('data-collapsed', newState);
+}
+
+collapsibles.forEach(
+  (collapsible) => {
+    collapsible.addEventListener('click', collapse);
+  }
+);
+
+// Authentication Section
+
+const tokenElm = document.querySelector('#access-token');
+const saveElm = document.querySelector('#access-token-save');
+const authStatusElm = document.querySelector('#access-token-status');
+const registerElm = document.querySelector('#registration-register');
+const unregisterElm = document.querySelector('#registration-unregister');
+const registrationStatusElm = document.querySelector('#registration-status');
+
+function saveCredentials() {
+  console.log('Authentication#saveCredentials()');
+
+  tokenElm.disabled = true;
+  saveElm.disabled = true;
+  authStatusElm.innerHTML = 'Saving...';
+
+  webex = window.webex = Webex.init({
+    credentials: {
+      access_token: tokenElm.value
+    }
+  });
+
+  webex.once('ready', () => {
+    console.log('Authentication#saveCredentials() :: Webex Ready');
+    registerElm.disabled = false;
+    authStatusElm.innerHTML = 'Saved';
+  });
+}
+
+function register() {
+  console.log('Authentication#register()');
+  registerElm.disabled = true;
+  registrationStatusElm.innerHTML = 'Registering...';
```

Comment:
Let's add a `unregisterElm.disabled = true;` here too so we don't attempt to unregister during our register.

Reply:
Don't need to do this, there is no way that the `unregister` button can be enabled without a registered meetings plugin. Let me know if you found a case that it is.

Reply:
Implemented anyways!